gs-rest-client/src/main/java/client/SimpleClient.java

```diff
@@ -90,11 +88,13 @@ public static String getLinesFromFile(String fileName, int firstLine, int lastLi
     * Sends a ClientFile object (file + error message) to the server
     * as a JSON string and processes its response.
     */
-    public static void fixMyBug(ServerRequest serverRequest, String method) {
-        fixMyBug(serverRequest, null, method);
+    public static void makeRequest(ServerRequest serverRequest, String method) {
```

Comment:
Rename method to be more generic and accurate. This method handles fixing bugs, echoing requests, and indexing by making a request to our server.